markdown_stoplight/changelog.md

```diff
@@ -2,164 +2,236 @@
 
 This change log covers significant updates and changes to the Beyond REST API.
 
+**2019-05-09**
+
+**Update**
+
+* Added `productIdentifiers` property to [Show variation details](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/products/variation-get) and [Update variation partially](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/products/variation-patch-json).
+
+* Added `workflow` property to [Show payment method details](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/payment-methods/payment-method-get).
+
+* Added query parameter `fileName` and header `Content-Type` to request parameters of [Add product image](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/products/product-images-create).
+
+* Added `bankName`, `iban`, `bic`, `bankAccount`, and `bankIdentifier` properties to [Show legal details](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/shop/merchant-legal-get) and [Update legal resource partially](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/shop/merchant-legal-jsonpatch) resources.
+
+**2019-04-23**
+
+**New**
+
+Add *Events* section. This section explains how applications can receive information about customer actions in ePages online shops.
+
+**2019-04-17**
+
+**New**
+
+* [Create script tag](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/script-tags/script-tags-create)
+
+* [List script tags](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/script-tags/script-tags-get-all)
+
+* [Show script tag details](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/script-tags/script-tags-get) 
+
+* [Update script tag](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/script-tags/script-tags-update)
+
+* [Delete script tag](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/script-tags/script-tags-delete)
+
+* [Create newsletter target](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/newsletter-target/newsletter-target-create)
+
+* [Show newsletter target details](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/newsletter-target/newsletter-target-get)
+
+* [Update newsletter target](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/newsletter-target/newsletter-target-update)
+
+* [Delete newsletter target](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/newsletter-target/newsletter-target-delete)
+
+**2019-03-08**
+
+**Deprecate**
+
+* Removed endpoint *List products by category*.
```

Comment:
Maybe it would be helpful to include the request path, e.g. `GET /product-view/products/search/find-by-category`